test/test_mathml_evaluation.py

```diff
@@ -1,22 +1,18 @@
+"""Test implementation of voltage clamps.
```

Comment:
I think some dodgy copy & paste has happened here in the past!

Reply:
I copied the docstring from the class to the module level. Turned it into comments now if you prefer?

Reply:
It wasn't that, it's that the docstring doesn't match what the test is doing :)

Reply:
Fair enough. I forgot to read :man_facepalming: